charts/openvasd/README.md

```diff
@@ -0,0 +1,46 @@
+Contains the helm chart to deploy openvasd.
+
+# Install
+
+To install openvasd helm chart from a local path execute
+
+``` 
+helm install openvasd ./openvasd/ -f openvasd/values.yaml
+```
+
+You can also provide override the initial values within `openvasd/values.yaml` by providing an additional `-f` flag.
+
+As an example imagine you want to override the openvas image to your forked one you can create `~/openvasd.yaml` file containing: 
+
+```
+# Contains openvasd
+openvas:
+  repository: nichtsfrei/openvas-scanner
+  pullPolicy: Always
+  tag: "edge"
+```
+
+if you then execute: 
+``` 
+helm install openvasd ./openvasd/ -f openvasd/values.yaml -f ~/openvasd.yaml
+```
+
+it will use `nichtsfrei/openvas-scanner` instead of `greenbone/openvas-scanner`.
+
+# Design decisions
+
+## OSPD and Redis via unix socket
+
+Although it is possible to start OSPD with TLS it is used in unix socket mode to prevent a user to bypass openvasd and interfere with those scans.
```

Comment:
```suggestion
Although it is possible to start OSPD with TLS, it is used in unix socket mode to prevent a user to bypass openvasd and interfere with those scans.
```